examples/bind-shopping.amp.html

```diff
@@ -0,0 +1,454 @@
+<!doctype html>
```

Comment:
> May I ask why? Angular uses double-bracing for string interp. while React doesn't.

I can be convinced that a single brace is enough. The only reservation I have is that if the probability of seeing a `{` in the natural content is `p`, then it will be `p^2` for `{{`. :)